tests/system/test_system.py

```diff
@@ -909,6 +942,17 @@ def test_third_level(self):
         self.assertIsNone(iterator.next_page_token)
         self.assertEqual(iterator.prefixes, set())
 
+    @RetryErrors(unittest.TestCase.failureException)
+    def test_include_trailing_delimiter(self):
+        iterator = self.bucket.list_blobs(
+            delimiter="/", include_trailing_delimiter=True
+        )
+        page = six.next(iterator.pages)
+        blobs = list(page)
+        self.assertEqual([blob.name for blob in blobs], ["file01.txt", "parent/"])
+        self.assertIsNone(iterator.next_page_token)
+        self.assertEqual(iterator.prefixes, set(["parent/"]))
```

Comment:
how is this request different when `include_trailinig_delimitier=False`? I don't see the difference between the system test in this case.

Reply:
The difference is that this is the only name that includes `parent/` in `self.assertEqual([blob.name for blob in blobs], ["file01.txt", "parent/"])`.

Reply:
I'll leave this for you to resolve if you're satisfied with that explanation